### DISCUSSION TITLE
ROLLBACK_FAILED on creating new stack (Hubs cloud personal).

### DISCUSSION THREAD

**Discussion by quansolashi** [mozilla/hubs#5762] [Hubs Cloud Questions]
I created a new Hubs cloud stack on AWS following [this tutorial](https://www.youtube.com/watch?v=W0erzlDpo0U). However, the below error occurs:

> The following resource(s) failed to delete: [SubnetBPrivate, AppDbSecurityGroup, SubnetAPrivate, AppSecurityGroup, AppDb, VPC, StreamSecurityGroup, AppDbSubnet].

![image](https://user-images.githubusercontent.com/71245792/197953229-f51b0f19-2c43-465e-8c0c-e22c56cbb3ce.png)

Can you tell me the cause and solution for this problem? Thank you.

**Comment by pattersonbl2** [comment]
It could be various reasons this error happened. The most common would be if you made changes in AWS that are not in the cloud formation.  You would have to got to each resource and delete by hand to clear this.
